### PR TITLE
Support c++ builds without extensions

### PR DESCRIPTION
**Pull request by onelxj**

### CODE REVIEW

File: tests/protoc-gen-sample.py [hunk 1]
```diff
@@ -53,6 +53,14 @@ def finalize(self) -> None:
...
             template_data['message_imports'] = '\n'.join(self._message_imports.values())
 
+            index = len(output_file_name)
+            for ch in reversed(output_file_name):
+                if (ch == '/'):
+                    break
+                index -= 1
+
+            output_file_name = output_file_name[index:]
+
```
Comment (while-false) [inline tests/protoc-gen-sample.py:63]: See comment in other repo/pr suggesting
```python
output_file_name = output_file_name.split('/')[-1]
```
;D

**Comment by while-false** [comment]
Hi @onelxj, happy to help you debug your issue. With a quick look at the code, my expectation is that the declared output directory is never being created, neither from bazel nor from python.

If I check out this branch, how can I best try it out/provoke the error you are seeing?

**Comment by benh** [comment]
@onelxj are you blocked here?

**Review by while-false — APPROVED** [review]
LGTM, a few comments, but mostly nit. :) Thanks @onelxj!

**Pull request merged**